tests/src/core/Transform.test.ts

```
@@ -59,4 +60,85 @@ describe("Transform test", function () {
+    state.scale ? state.scale.copyFrom(this.scale) : (state.scale = this.scale.clone());
+    state.size ? state.size.copyFrom(this.size) : (state.size = this.size.clone());
+  }
+
+  /**
+   * @internal
+   */
+  _applyTransitionState(state: SubClassOfTransformTransitionState): void {
+    this.position.copyFrom(state.position);
+    this.rotation.copyFrom(state.rotation);
+    this.scale.copyFrom(state.scale);
+    this.size && this.size.copyFrom(state.size);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)